robolectric/src/test/java/org/robolectric/RobolectricTestRunnerSelfTest.java

```diff
@@ -107,6 +107,11 @@ public void testVersionConfiguration() {
         .isEqualTo("4.4_r1");
   }
 
+
+  @Test public void assertThatz() throws Exception {
```

Comment:
Want to do this more? I'm guessing that it is checking that things are loaded from the right classloader, but I'm a little confused as its an assert for a primitive rather than an object?